website/static/js/profile.js

```diff
@@ -693,10 +731,19 @@ var JobViewModel = function() {
     DateMixin.call(self);
     TrackedMixin.call(self);
 
-    self.institution = ko.observable('').extend({required: true, trimmed: true});
     self.department = ko.observable('').extend({trimmed: true});
     self.title = ko.observable('').extend({trimmed: true});
 
+    self.institution = ko.observable('').extend({
+        trimmed: true,
+        required: {
+            onlyIf: function() {
+               return !!self.department() || !!self.title();
```

Comment:
For some reason, a :exclamation: does not work with knockout require, so :bangbang: are back in.